plugins/win-capture/data/compatibility.json

```diff
@@ -17,8 +17,8 @@
             "url": "https://help.steampowered.com/faqs/view/09A0-4879-4353-EF95#whitelist"
         },
         {
-            "name": "Electron",
-            "translation_key": "Compatibility.Application.ElectronBitBlt",
+            "name": "Chromium",
+            "translation_key": "Compatibility.Application.ChromiumBitBlt",
```

Comment:
Does this actually affect Chromium? Either way I'd prefer if you kept the existing translation key and just updated the text to say "Electron/Chromium" instead.

Reply:
`Electron` includes `Chromium`. `Edge` and `Chrome` are also recognized as `Electron`, but they are not built on `Electron`.
https://wikipedia.org/wiki/Electron_(software_framework)

That's why I changed `Electron` to `Chromium`.

Like Steam, it can actually capture frames, but cannot capture pages. Because their sub windows use `Chromium`.
![QQ截图20230402213709](https://user-images.githubusercontent.com/26628474/229356380-f2ab6f85-ee1f-483c-a900-971636b19825.png)

Reply:
While Electron is using Chromium, the average user is more familiar with Electron than Chromium. Renaming it to "Electron/Chromium" and retaining the current translation key is probably the best way forward for clarity and translation workflow.

My question was more about whether Chromium/CEF based applications even use the `Chrome_WidgetWin_0` window class.